sanic/blueprints.py

```diff
@@ -148,17 +149,33 @@ def signal(self, event: str, *args, **kwargs):
     def copy(
         self,
         name: str,
-        url_prefix: str = "",
-        version: Optional[Union[int, str, float]] = None,
-        strict_slashes: Optional[bool] = None,
-        version_prefix: str = "/v",
+        url_prefix: str = _default,
+        version: Optional[Union[int, str, float]] = _default,
+        version_prefix: str = _default,
+        strict_slashes: Optional[bool] = _default,
     ):
+        """
+        Copy a blueprint instance with some optional parameters to 
+        override the values of attributes in the old instance.
+
+        :param name: unique name of the blueprint
+        :param url_prefix: URL to be prefixed before all route URLs
+        :param host: IP Address of FQDN for the sanic server to use.
+        :param version: Blueprint Version
+        :param version_prefix: the prefix of the version number shown in the URL.
+        :param strict_slashes: Enforce the API urls are requested with a
+            training */*
```

Comment:
typo: "trailing"

This is technically only part of `strict_slashes`. It enforces the path matching will meet the existence or non-existence of the trailing slash as defined.